### DISCUSSION TITLE
only one instance of SMPlayer not working

### DISCUSSION THREAD

**Discussion by michaelbr51** [smplayer-dev/smplayer#333] [General]
I have the following version of SMPlayer installed on a MX Linux 19.4.
Version: 21.10.0 (revision 10000)
Using Qt 5.12.7 (compiled with Qt 5.12.7)
Using MPlayer SVN r38132
Until last month, it was working fine, now the only one instance is not working (no matter if I check the box in Options > Preferences > Interface > Instances > Use only one instance of running SMPlayer, it ALWAYS runs on multiple instances), is there any other place that I can force only one instance?
ps: the smplayer.ini file shows `single_instance_enabled=true`.

**Comment by smplayer-dev** [comment]
Please send the smplayer log.

**Reply by michaelbr51** [reply]
Mplayer/mpv log is empty.
[SMPlayer_log.txt](https://github.com/smplayer-dev/smplayer/files/7777426/SMPlayer_log.txt)

**Comment by smplayer-dev** [comment]
It seems you're using the flatpak package. I've just tested it and the one instance option works fine for me with this package.
Be sure you don't have another version of smplayer installed as well (from a deb or rpm package).

**Reply by michaelbr51** [reply]
Thanks for this tip, you're right, I was using a stable one, then installed the flatpack, when I noticed this problem, tried to remove the flatpack and install the stable one, but it won't work, how can I troubleshoot where is the problem (the app won't start, after the install, it said installed successfully, but when I clicked it, nothing shows up, task manager shows smplayer is running, but no UI). I used package manager to un/install, how can I make sure that uninstall was complete (everything has been removed?).

**Reply by smplayer-dev** [reply]
Run smplayer from a terminal and see if there are error messages.

**Reply by michaelbr51** [reply]
after few minutes got this error, please see the command
```
$ smplayer
This is SMPlayer v. 21.10.0 (revision 10000) running on Linux
ICE default IO error handler doing an exit(), pid = 19917, errno = 32
```

**Reply by smplayer-dev** [reply]
A google search indicates that this error might be fixed deleting the file ~/.ICEauthority

**Reply by michaelbr51** [reply]
deleted the file as suggested, now got another error, as follows
```
$ smplayer
Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
This is SMPlayer v. 21.10.0 (revision 10000) running on Linux
```

**Reply by smplayer-dev** [reply]
I don't know, maybe a permissions problem.
Is this happening with the deb package or the flatpak package?

**Reply by michaelbr51** [reply]
thanks so much for spending time to try to solve my problem, using flatpak SMPlayer works fine, the only issue so far is only one instance not working, with package manager (deb package), it installed OK, but can't start, the weird  thing is both has the same version! Maybe my OS is messed up, I'll try to re-install it. Thanks for your time.

**Reply by 1manBANG** [reply]
Have you solved this issue? Everybody else has this problem too? I need only one window open and video playing inside SMPlayer, so after OS reinstall this player cannot do it, so it is not usable for me. 

Advanced settings change ''Run Mplayer/mpv in it's own window'' is not respected by chech/uncheck.

**Reply by michaelbr51** [reply]
yes, it's solved, my guess is my OS was messed up (I've tried several different apps, installing & removing), after a fresh install, everything is fine now. Please note that I'm using SMPlayer and not MPlayer, not sure if this makes any difference.
